test-clients/src/main/java/com/hedera/services/yahcli/commands/signedstate/00-running-yahcli-to-get-contract-info.md

```diff
@@ -0,0 +1,47 @@
+- (all examples using a signed state file from ~2022-11-10)
```

Comment:
This would be a good location instead to get an intro and overview of the tool command
```suggestion
# Yahcli signedstate 

The signedstate command offers ....
```

Reply:
Please review the first part of this doc again and to see if it is now suitable.